OADB/EMCAL/READMEoadb.txt

```diff
@@ -40,5 +40,6 @@ In addition, a short history of changes to the files in EOS will be listed here:
 - 20180618: Update of EMCALBadChannels.root with runwise maps for LHC18d (except for runs 286129 286350 286341 286313)
 - 20180621: Update of EMCALTimeCalib.root and EMCALTimeL1PhaseCalib.root with LHC16o and LHC16p time calibrations
 - 20180622: Update of EMCALBadChannels.root with additional cells for LHC16j and LHC16l and a fix for the LHC16h maps which were not properly added in OADB before
+- 20180706: Update of EMCALBadChannels.root with additional cells for LHC16p and update of EMCALTimeCalib.root with updated calibrations for LHC17o
```

Comment:
Do we have ticket IDs here?

Reply:
Not for the updates. But I could mention the original calibration tickets in the PRs description.

Reply:
Tickets are mentioned in the PR description. I would be thankful for a +1.